### PR TITLE
Fixing: selector-class-pattern and selector-id-pattern SCSS interpolation

### PR DESCRIPTION
**Pull request by alan-agius4**
Ref #988

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 - Added: exposed `stylelint.createRuleTester()`.
 - Fixed: `property-value-blacklist` and `-whitelist` no longer error on properties without a corresponding list entry.
 - Fixed: `font-weight-notation` now ignores `initial` value.
+- Fixed: `selector-id-pattern` no longer warns about Sass interpolation when an Id is specified through interpolation
```
Comment (davidtheclark) [inline CHANGELOG.md]: I'd be more direct here: "`selector-id-pattern` ignores selectors with Sass interpolation.

**Comment by davidtheclark** [comment]
Looks good. Thanks for contributing! 

It needs to be synced up with master, and then we'll see if the tests pass.

**Comment by jeddy3** [comment]
@alan-agius4 Can you *rebase* to `master` please, so we can see if the tests pass.

**Comment by alan-agius4** [comment]
Ill try to do it later on today. As i don't have access to a pc

**Comment by alan-agius4** [comment]
rebase done

**Comment by jeddy3** [comment]
Thank you.

Lgtm. @davidtheclark Feel free to merge if you're happy with it.

**Pull request merged**